profiles/common/modules/features/nexteuropa_dgt_connector/tmgmt_poetry/tmgmt_poetry.module

```diff
@@ -284,34 +284,47 @@ function _tmgmt_poetry_form_load_translator_settings(&$form_state, &$form, $job)
  * Job Form Callback.
  */
 function _tmgmt_poetry_form_tmgmt_job_submit($form, &$form_state) {
+  // Current implementation part.
```

Comment:
In my opinion, we shouldn't change anything in the tmgmt_module and provide all overrides in a separated feature.